avro/src/avro_consumer.py

```diff
@@ -0,0 +1,26 @@
+from consumer_factory import ConsumerFactory
+from confluent_kafka import Consumer, KafkaException
+from config import get_config
+from serialize_avro import AvroSerializer
+from read_schema import get_schema
+
+
+def main(topic: str, serializer: AvroSerializer, consumer: Consumer) -> None:
+    consumer.subscribe([topic])
+    while True:
+        msg = consumer.poll(timeout=1.0)
+        if msg is None:
+            continue
+        if msg.error():
+            print(msg.error())
+        else:
+            msg = msg.value()
+            print(serializer.deserialize(msg))
+
+
+if __name__ == "__main__":
+    conf = get_config('config.ini')
+    con_obj = ConsumerFactory(conf['consumer']['group_id']).get_consumer()
+    schema = get_schema('user.avsc')
```

Comment:
What is get_schema doing ? read schema from file and return schema object ?

Reply:
found mistake in serializer class it is not serializing into avro it is serializing into byte stream will fix it now.
get schema is reading schema from the file

Reply:
it is correct babai, don't know anything about but the methods just copied them ,
the method just require the schema in json form. Merging now

Reply:
yes reads schema and returns dict object